pilot/pkg/config/kube/crdclient/types.gen.go

```diff
@@ -47,56 +48,67 @@ func create(ic versionedclient.Interface, sc serviceapisclient.Interface, cfg co
 		return ic.NetworkingV1alpha3().DestinationRules(cfg.Namespace).Create(context.TODO(), &clientnetworkingv1alpha3.DestinationRule{
 			ObjectMeta: objMeta,
 			Spec:       *(cfg.Spec.(*networkingv1alpha3.DestinationRule)),
+			Status:     *(cfg.Status.(*v1alpha1.IstioStatus)),
```

Comment:
I kind of remember `status` will be ignored when create, can you confirm with it.

Reply:
Found one specification: https://book-v1.book.kubebuilder.io/basics/status_subresource.html

Reply:
yeah great point. only on update() it can be changed.